src/main_mod_server.erl

```diff
@@ -0,0 +1,49 @@
+-module(main_mod_server).
```

Comment:
Same thing here with a prefix to avoid namespace clash.

Reply:
OK,  prefixed files and code reference with xqerl